src/test/java/com/cydeo/step_definitions/LoginPageStepDefs.java

```diff
@@ -15,19 +15,21 @@ public class LoginPageStepDefs {
 
         @Given("user is on the login page")
         public void userIsOnTheLoginPage() {
-            System.out.println("DONE IN HOOKS");
```

Comment:
Line 18 and 19 are still showing conflicts even though they are commented out. Can you delete these (maybe screenshot them first just in case).